setup-env

```diff
@@ -285,5 +282,8 @@ else:
 END_OF_LINE
 )"
 
+# Install all necessary mypy type stubs
+mypy --install-types --non-interactive src/
+
```

Comment:
Why did you move this code?

Reply:
If I recall correctly... if `mypy` encountered any missing stubs when running, it would prevent the pre-commit hooks from installing.  It was far from clear that this was happening.  This lead to local repository that claimed it had successfully initialized but in-fact had not. 

The change to using `--non-interactive` and placing the call to `mypy` after the installation of `pre-commit` hooks prevents this case from happening.

Reply:
Is this something that should be backported to the skeleton?

Reply:
Yes, I think it should be.

Reply:
We already have [this PR](https://github.com/cisagov/skeleton-python-library/pull/138) in progress - could be added in there.

Reply:
See [this commit](https://github.com/cisagov/skeleton-python-library/pull/138/commits/48d3b4f913657570e43dbaffbc80bbd604bfc5ad) in [the PR @dav3r mentioned](https://github.com/cisagov/skeleton-python-library/pull/138).